### PR TITLE
Dependabot: add dependabot.yaml

### PR DESCRIPTION
**Pull request by lzaoral**
This should enable Dependabot so that it will automatically create PRs with CI or git submodule related updates, run the build and test-suite.  So that we only have to merge these changes (if the CI will be green).

Depends on #227

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gitsubmodule"
+    # Files stored in repository root
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (lzaoral) [inline .github/dependabot.yml:7]: We might want to check the changes only weekly or monthly so that Dependabot does not generate too much noise.

Comment (kdudka) [reply .github/dependabot.yml:7]: Good point.  Monthly sounds like a good tradeoff.

Comment (kdudka) [reply .github/dependabot.yml:7]: Please ignore my previous comment.  I though I was commenting on a https://github.com/csutils/cscppc pull request.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - "trtikm"
+
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "lzaoral"
```
Comment (lzaoral) [inline .github/dependabot.yml:18]: Whoever decides to complete this PR should change this field to someone else.

**Pull request closed without merging**